bitnami/kube-prometheus/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: kube-prometheus
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/kube-prometheus
-version: 8.14.2
+version: 8.14.3
```

As it is something new, it is better to raise to minor
```suggestion
version: 8.15.0
```